public/app/plugins/datasource/cloudwatch/api.ts

```diff
@@ -1,4 +1,4 @@
-import { DataSourceInstanceSettings, SelectableValue } from '@grafana/data';
+import { DataSourceInstanceSettings, SelectableValue, toOption } from '@grafana/data';
```

Comment:
I may have missed that somewhere --Can you tell us more about the changes in this file? `toOption`, the change from `resourceRequest` to `getBackendSrv`

Reply:
`resourceRequest` is suitable for as-is resource requests that returns `[]suggestData`. This new api request returns data in a different format, so it needs to do its own mapping. `getDimensionKeys` now returns an array of strings, and `toOption` converts it to a selectable value suitable for selects.